### ISSUE TITLE
Does the Classical Image Super-Resolution have color enhancement function?

### ISSUE THREAD

**Issue by hcleung3325** [JingyunLiang/SwinIR#26]
I trained the Classical Image Super-Resolution with other training sets.
When I tested the model, I discovered that the color has changed to be sharper.
May I know that is this Classical Image Super-Resolution has color enhancement?

![swinir](https://user-images.githubusercontent.com/50704185/134465470-283f5a8a-e213-4519-b97f-b82ab3548d9f.png)
The left-hand side is swinir result.

**Comment by JingyunLiang** [comment]
No. It is not used in classical image SR, which follows the same settings as existing papers in Table 2.

What is your right-hand figure? Is it a LR image? What SwinIR does is to make LR images be sharper and clearer. You should compare with the ground-truth HR image to see if there is "color shift".
